src/create-config.ts

```diff
@@ -90,38 +145,19 @@ const main = async (operationRepositoryTarget?: string) => {
     };
   });
 
-  const secretAppend =
-    sanitiseFilename(apiData.apiMetadata.name).toUpperCase() + `_${cloudProviderTypeAWS.toUpperCase()}`;
+  const nodeSettingsAWS = buildNodeSettings(
+    apiData.apiMetadata.name,
+    cloudProviderTypeAWS,
+    cloudProviderRegionAWS,
+    response.airnodeHeartbeat
+  );
 
-  const nodeSettings = {
-    nodeVersion: require('@api3/airnode-node/package.json').version,
-    cloudProvider: {
-      type: cloudProviderTypeAWS,
-      region: cloudProviderRegionAWS,
-      disableConcurrencyReservations: false,
-    },
-    airnodeWalletMnemonic: '${AIRNODE_WALLET_MNEMONIC}',
-    heartbeat: {
-      enabled: response.airnodeHeartbeat,
-      ...(response.airnodeHeartbeat && {
-        apiKey: `\${HEARTBEAT_KEY_${secretAppend}}`,
-        id: `\${HEARTBEAT_ID_${secretAppend}}`,
-        url: `\${HEARTBEAT_URL_${secretAppend}}`,
-      }),
-    },
-    httpGateway: {
-      enabled: false,
-    },
-    httpSignedDataGateway: {
-      enabled: true,
-      apiKey: `\${HTTP_SIGNED_DATA_GATEWAY_KEY_${secretAppend}}`,
-      maxConcurrency: 200,
-    },
-    logFormat: 'plain' as const,
-    logLevel: 'INFO' as const,
-    stage: 'dev',
-    skipValidation: true,
-  };
+  const nodeSettingsGCP = buildNodeSettings(
+    apiData.apiMetadata.name,
+    cloudProviderTypeGCP,
+    cloudProviderRegionGCP,
+    response.airnodeHeartbeat
+  );
```

Comment:
I really like that this got extracted into its own function but shouldn't we call buildNodeSettings() for gcp only when response.gcp is true?

Reply:
I was thinking of implementing the above but the current implementation seems simple and clean. The object only gets appended if response.gcp is true . adding if statements or `&&` seemed redundant.

Reply:
The thing is that if I call this script to create the config just for AWS and I get errors because of creating unnecessary GCP objects then that would be a bad user experience I think (borderline annoying).

 If this is blocking something then just merge it and I'll try to refactor it later.